code/go/0chain.net/blobbercore/allocation/allocationchange.go

```diff
@@ -161,7 +165,9 @@ func (a *AllocationChangeCollector) CommitToFileStore(ctx context.Context) error
 
 func (a *AllocationChangeCollector) DeleteChanges(ctx context.Context) error {
 	for _, change := range a.AllocationChanges {
-		change.DeleteTempFile()
+		if err := change.DeleteTempFile(); err != nil {
+			logging.Logger.Error("AllocationChangeProcessor_DeleteTempFile", zap.Error(err))
```

Comment:
not sure if it makes more sense to break the loop and return an error here, since it will be a departure from the current flow.